composer.json

```diff
@@ -57,8 +56,7 @@
         "symfony/cache": "^6.4||^7.0"
     },
     "conflict": {
-        "doctrine/lexer": "<1.0||>=3.0",
-        "doctrine/orm": "<2.8||>=3.0"
```

Comment:
It will be good to enrich the CI steps with an updated [matrix](https://github.com/martin-georgiev/postgresql-for-doctrine/blob/main/.github/workflows/ci.yml#L20) that explicitly checks both ORM v2 and ORM v3 will run the test suite without issues. For example, the setup may look like the below pseudocode (I haven't run it so, take it with a pinch of salt):
```shell
#.github/workflows/ci.yml

jobs:
  tests:
    name: "PHP ${{ matrix.php }} + Doctrine ORM ${{ matrix.doctrine-orm }} + Composer ${{ matrix.composer-flags }}"
    runs-on: ubuntu-latest

    strategy:
      fail-fast: false
      matrix:
        include:
          - php: '8.1'
          - php: '8.2'
          - php: '8.3'
            ignore-php-version: true
            calculate-code-coverage: true
          - doctrine-orm: '2.8'
          - doctrine-orm: '3.0'
          - doctrine-orm: 'latest'

    - name: Install Doctrine ORM dependencies
      run: |
        if [ "${{ matrix.doctrine-orm }}" == "2.8" ]; then
          composer require doctrine/orm "~2.8" --prefer-dist --no-interaction --no-progress ${{ matrix.composer-flags }}
        elif [ "${{ matrix.doctrine-orm }}" == "3.0" ]; then
          composer require doctrine/orm "~3.0" --prefer-dist --no-interaction --no-progress ${{ matrix.composer-flags }}
        else
          composer update --prefer-dist --no-interaction --no-progress ${{ matrix.composer-flags }}
        fi